src/wallets/helpers.ts

```diff
@@ -1,20 +1,18 @@
 import { LOCAL_STORAGE_SELECTED_WALLET_ID } from "../constants";
 import { updateState } from "../state/State";
+import { storage } from "../services/persistent-storage.service";
 
 export const setSelectedWalletId = (walletId: string | null) => {
   if (walletId) {
-    localStorage.setItem(
-      LOCAL_STORAGE_SELECTED_WALLET_ID,
-      JSON.stringify(walletId)
-    );
+    storage.setItem(LOCAL_STORAGE_SELECTED_WALLET_ID, JSON.stringify(walletId));
```

Comment:
Just tested this and it looks like we're doubling up on the prefix. I noticed the key has changed to `near-wallet-selector-near-wallet-selector:selectedWalletId`. This is happening for all the keys